src/critnib/critnib.c

```diff
@@ -162,7 +168,7 @@ static inline unsigned slice_index(word key, sh_t shift) {
 /*
  * critnib_new -- allocates a new critnib structure
  */
-struct critnib *critnib_new(void) {
+struct critnib *critnib_new(free_leaf_t cb_free_leaf, void *leaf_allocator) {
```

Comment:
please add a description of args in the comment above

Reply:
Done